openshift/hac-dev
# create component with git context and revision

## Fixes 
[
](https://issues.redhat.com/browse/HAC-2743)

## Description
Git reference (as revision) and context were not set for CDQ and component creation

## Type of change




## How to test or reproduce?
Create a monorepo such that a git context can be provided and also ensure there is a valid branch on the repo to test with.
When creating a component, set the git reference and context appropriately.
Create the component.

Note that due to a bug in HAS, the revision is returned in the component stub response of CDQ. As such the Component is still missing the revision at creation time. This should get resolved when the HAS bug is addressed

## Browser conformance:
